src/go/rpk/pkg/cli/version/version.go

```diff
@@ -131,7 +133,9 @@ func printClusterVersions(rpv *redpandaVersions) {
 	if len(*rpv) == 0 {
 		fmt.Println(`  Unreachable, to debug, use the '-v' flag. To get the broker versions, pass the
   hosts via flags, profile, or environment variables:
-    rpk version -X admin.hosts=<host address>`)
+    rpk version -X admin.hosts=<host address>
+
+  To get only the rpk version, use 'rpk --version'.`)
```

Comment:
Is it worth mentioning you can also use -v?

Reply:
We use `-v` for `verbose`. We have no shorthands for `--version`

Reply:
Oh that's right 🤦‍♀️